### PR TITLE
Be clear that data_base64 isn't a CE attribute

### PR DESCRIPTION
**Pull request by duglin**
Signed-off-by: Doug Davis <dug@microsoft.com>

### CODE REVIEW

**Comment by duglin** [comment]
@lance did you have an alternative wording you'd like me to use?

**Review by jskeet — APPROVED** [review]

File: cloudevents/formats/json-format.md [hunk 1]
```diff
@@ -144,6 +144,10 @@ not provided, no assumptions can be made as to the format of the data and
 therefore the `datacontenttype` attribute MUST NOT be present in the resulting
 CloudEvent.
 
+Note: Definition of `data_base64` is a JSON-specific formatting rule and not
```
Comment (lance) [inline cloudevents/formats/json-format.md:147]:
```suggestion
Note: Definition of `data_base64` is a JSON-specific marshaling rule and not
```
@duglin not sure if this is much better, but I think it is slightly more accurate.

Comment (duglin) [reply cloudevents/formats/json-format.md]: done!

**Comment by duglin** [comment]
Approved on the 7/21 call

**Pull request merged**